### PR TITLE
📩🤏 Update import illustration

### LINKED ISSUE #1211: Import account
### Discord Discussion Link

_No response_

### What browsers are you seeing the problem on?

Chrome

### What were you trying to do?

Import an account

### What did not work?

Functionality worked, but illustration for importing accounts has reverted back to old version.

image

it should be this one

image

### Version

Community Edition v0.12.2

### Relevant log output

_No response_

### PR DESCRIPTION
**Pull request by henryboldi**
Using a png here since the illustration design is somewhat complex and uses shadows. Moving away from the MetaMask alluding imagery.

<br/>
import3

Closes #1211

### CODE REVIEW

**Review by VladUXUI — COMMENTED** [review]
Looking good

File: ui/pages/Onboarding/OnboardingImportMetamask.tsx [hunk 1]
```diff
@@ -227,12 +227,12 @@ export default function OnboardingImportMetamask(props: Props): ReactElement {
...
           margin-top: ${HIDE_IMPORT_DERIVATION_PATH ? "35px" : "24px"};
         }
-        .metamask_onboarding_image {
+        .illustration_import {
           background: url("./images/illustration_import_seed@2x.png");
           background-size: cover;
-          width: ${HIDE_IMPORT_DERIVATION_PATH ? "205.3px" : "155px"};
-          height: ${HIDE_IMPORT_DERIVATION_PATH ? "193px" : "145.71"};
-          margin-top: ${HIDE_IMPORT_DERIVATION_PATH ? "27px" : "23.5px"};
+          width: 106.5px;
```
Comment (jagodarybacka) [inline ui/pages/Onboarding/OnboardingImportMetamask.tsx:233]: Do we need to be so precise with these `0.5px` 😄 ?

**Review by jagodarybacka — APPROVED** [review]
Looks good ✨ 

Non blocking questions:

Do we plan to change metamask icon here as well? @VladUXUI 
image

Also text `How do I find...` suggests it should be clickable but it isn't - do we have existing issue for this bug?
image

File: ui/pages/Onboarding/OnboardingImportMetamask.tsx (hunk 1 above)
Comment (VladUXUI) [reply ui/pages/Onboarding/OnboardingImportMetamask.tsx:233]: >Do we plan to change metamask icon here as well? @VladUXUI

Yes, this entire page is getting a remake and Henry has another PR for this.


> Also text How do I find... suggests it should be clickable but it isn't - do we have existing issue for this bug?

I don't think this is ticketed

Comment (jagodarybacka) [reply ui/pages/Onboarding/OnboardingImportMetamask.tsx:233]: Ok, I will make issue for it.

**Comment by jagodarybacka** [comment]
Let's merge it 🚀

**Pull request merged**